src/project-css/app/Helpers/TableHelper.php

```diff
@@ -0,0 +1,555 @@
+<?php
+/**
+ * @author Ajay Krishna Teja Kavur
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+
+use App\Jobs\CreateSearchIndex;
+use App\Jobs\FileImport;
+use App\Jobs\OptimizeSearchIndex;
+use App\Jobs\UpdateSearchIndex;
+
+use App\Models\Collection;
+use App\Models\CMSRecords;
+use App\Models\Table;
+use App\Helpers\CSVHelper;
+use App\Helpers\CMSHelper;
+use App\Helpers\CustomStringHelper;
+
+class TableHelper {
+    /**
+     * Table Helper
+     *
+     * These are various functions that help with dynamically
+     * creating and modifying tables without migrations.
+     *
+     */
+
+    /** 
+    * Inserts New String field into the passed table
+    *
+    * @param table $table - Current Table object
+    * @param string $curColNme - String name of new field
+    * @param string $curColSze - String Size either 'default', 'medium' or 'big'
+    * @author Tracy A McCormick
+    */     
+    public function createStringField($table, $curColNme, $curColSze) {
```

Comment:
This appears to be duplicate, you are doing this in another class also with a big switch statement.